cluster/gce/debian/master-helper.sh

```diff
@@ -40,8 +40,31 @@ function create-master-instance {
 
   write-master-env
   prepare-startup-script
-  gcloud compute instances create "${MASTER_NAME}" \
-    ${address_opt} \
+  create-master-instance-internal "${MASTER_NAME}" "${address_opt}" "${preemptible_master}"
+}
+
+function replicate-master-instance() {
+  local existing_master_zone="${1}"
+  local existing_master_name="${2}"
+  local existing_master_replicas="${3}"
+
+  local kube_env="$(get-metadata "${existing_master_zone}" "${existing_master_name}" kube-env)"
+  # Substitute KUBERNETES_MASTER_NAME to the replica's name.
+  kube_env="$(echo "${kube_env}" | grep -v "KUBERNETES_MASTER_NAME")"
+  kube_env="$(echo -e "${kube_env}\nKUBERNETES_MASTER_NAME: '${REPLICA_NAME}'")"
```

Comment:
Just ensuring - it is excpect that you are replacing this here and you're not doing it in GCI, right?